doc/ko/INSTALL.md

```diff
@@ -0,0 +1,188 @@
+QUICK GUIDE
+===========
+
+리눅스 배포판에서는 다음의 명령어들을 사용하여 uftrace를 빌드하고 설치할 수 있습니다.
+
+    $ sudo misc/install-deps.sh    # optional for advanced features
+    $ ./configure                  # --prefix can be used to change install dir
+    $ make
+    $ sudo make install
+
+더 자세한 사항은 아래의 내용을 참고하십시오.
+
+
+GETTING THE SOURCE
+==================
+최신 버전의 uftrace는 Github에서 사용할 수 있습니다.
+
+  https://github.com/namhyung/uftrace
+
+
+DEPENDENCY
+==========
+
+uftrace는 C언어로 작성되었고, 외부 의존성을 최소화하였습니다.
+현재, utrace는 외부 라이브러리없이 빌드 가능합니다. 하지만 더 많은 고급 기능들을 사용하기 위해서는 아래의 라이브러리들을 설치하는 것이 좋습니다.
+
+먼저, uftrace의 종속성을 제대로 감지할 수 있도록하는 `pkg-config`가 시스템에 설치되어 있는지 확인해야 합니다.
+만약, 그렇지 않다면 이미 설치되어 있어도 일부 패키지가 감지되지 않아 uftrace의 일부 기능이 비활성화될 수 있습니다.
+
+과거에는 uftrace가 ELF 파일 조작을 위해 elfutils 프로젝트의 `libelf`에 의존했습니다.
+더 이상 `libelf`가 필수는 아니지만 ELF 실행파일을 더 잘 처리하기 위해서는 설치하는 것을 권장합니다.
+또한, DWARF 디버그 정보를 처리하기 위해서는 `libdw` 라이브러리를 설치하는 것이 좋습니다.
+libdw 자체는 `libelf`에 의존하므로 `libdw`만 설치하면됩니다.
+
+debian기반의 시스템에서는(예: Ubuntu) `libdw-dev` 패키지가 필요한 라이브러리 및 파일들을 제공해줍니다.
+
+    $ sudo apt-get install libdw-dev
+
+redhat기반의 시스템에서는(예: Fedora, RHEL) `elfutils-devel`이 필요합니다.
+
+    $ sudo dnf install elfutils-devel
+
+또한, uftrace에서는 libstdc++ 라이브러리를 사용하여 C++ symbol들을 완전하게 demangling합니다.
+하지만 uftrace는 더 짧은 symbol name을 위한 자체 demangler를 가지고 있으므로 libstdc++ 라이브러리가 필수는 아닙니다.
+(uftrace의 자체 demanlger는 인수와 템플릿 등을 생략합니다.)
+
+ncursesw 라이브러리는 콘솔에서 테스트 사용자 인터페이스(TUI)를 구현하는데 사용됩니다.
+ncurses(w) 라이브러리는 터미널 처리 루틴 위에서 빌드된 `uftrace tui` 명령어를 제공합니다.
+이는 트레이스 데이터 분석의 사용자 경험을 향상시켜주므로 만약, `uftrace graph`나 `uftrace report`와 같은 작업을 자주 수행하는 경우 설치하는 것이 좋습니다.
+
+또한, 마크다운 문서에서 man 페이지를 빌드하려면 `pandoc`이 필요합니다.
+
+
+BUILD
+=====
+
+uftrace를 빌드하려면 먼저, gcc 및 make와 같은 기본 소프트웨어 개발 도구들을 설치해야 합니다. 그리고 종속성있는 패키지들도 함께 설치해야 합니다.
+자세한 내용은 DEPENDENCY 부분을 참조하십시오.
+
+필요한 소프트웨어들을 설치한 후에는 `configure`를 실행하여 설치 경로 및 기타 기능들을 설정해야 합니다. `configure`실행 시 디폴트도 /usr/local 경로에 uftrace를 설치합니다.
+만약, uftrace를 다른 위치에 설치하려면 `prefix`변수를 설정해주면 됩니다. (아래를 참조)
+
+    $ ./configure --prefix=/usr
+
+prefix 경로와 감지된 기능들이 다음과 같이 표시됩니다:
+
+    uftrace detected system features:
+    ...         prefix: /usr
+    ...         libelf: [ on  ] - more flexible ELF data handling
+    ...          libdw: [ on  ] - DWARF debug info support
+    ...      libpython: [ on  ] - python scripting support
+    ...      libluajit: [ OFF ] - luajit scripting support
+    ...    libncursesw: [ on  ] - TUI support
+    ...   cxa_demangle: [ on  ] - full demangler support with libstdc++
+    ...     perf_event: [ on  ] - perf (PMU) event support
+    ...       schedule: [ on  ] - scheduler event support
+    ...       capstone: [ on  ] - full dynamic tracing support
+    ...      libunwind: [ OFF ] - stacktrace support (optional for debugging)
+
+그런 다음 `make`를 실행하여 소스파일을 빌드할 수 있습니다.
+
+    $ make
+
+다음을 통해 uftrace를 빌드하여 결과 실행파일을 현재 경로에 배치할 수 있습니다.
+이는 테스트 용도로도 좋지만 일반적인 사용을 위해서도 설치할 수 있습니다.
```

Comment:
In the English version, the line looked like this.
> It builds uftrace, placing the resulting binaries in the current directory. This is good for testing, but you'll want to install it for normal use.


so this is just a suggestion, but how about this?

```
84: 위의 명령어는 uftrace를 빌드하며, 명령어 실행 결과로 uftrace 실행파일을 현재 경로에 생성합니다. 
85: 이 단계까지는 테스트 목적으로 나쁘지 않지만, 일반적인 사용을 위해서는 아래 명령어로 설치하는 것이 좋습니다.
```

Reply:
Thank you for review of my translation.
I'll fix this part as you said. :smiling_face_with_3_hearts: